examples/hello_canvas.rs

```diff
@@ -57,7 +57,7 @@ impl event::EventHandler<ggez::GameError> for MainState {
 
             canvas.draw_text(
                 &[text.color(Color::from((0, 0, 0, 255)))],
-                dest_point - vec2(15., 15.),
+                dest_point + vec2(15., 15.),
```

Comment:
What's this for?

Reply:
Ah, nothing big, it just looks a bit nicer when the text gets displaced towards the center, instead of towards the edge.